### PR TITLE
feat: update for k8s 1.18

### PR DESCRIPTION
**Pull request by danieltanfh95**
I've created a new branch to use this for the latest version of k8s.

Tested to work on AKS without much fanfare. Just needed to remove the previous nvidia gpu plugin.

### CODE REVIEW

File: deepomatic-shared-gpu-nvidia-device-plugin.yml [hunk 1]
```diff
@@ -1,9 +1,14 @@
-apiVersion: extensions/v1beta1
+apiVersion: apps/v1
 kind: DaemonSet
 metadata:
   name: deepomatic-shared-gpu-nvidia-device-plugin-daemonset
-  namespace: kube-system
+  namespace: gpu-resources
```
Comment (thomas-riccardi) [inline deepomatic-shared-gpu-nvidia-device-plugin.yml:5]: let's keep the current namespace: `kube-system` (upstream still uses `kube-system` too)

**Review by thomas-riccardi — COMMENTED** [review]
Thank you for the PR!
I just have one remark about the namespace change.

File: deepomatic-shared-gpu-nvidia-device-plugin.yml (hunk 1 above)
Comment (danieltanfh95) [reply deepomatic-shared-gpu-nvidia-device-plugin.yml:5]: Sure, let's use kube-system

**Comment by thomas-riccardi** [comment]
thanks @danieltanfh95 !

**Pull request merged**